openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -436,6 +436,13 @@ def get_registration_form(self, request):
                 if field['name'] == 'confirm_email':
                     del form_desc.fields[index]
                     break
+        is_invited = bool(request.GET.get("is_invited", None))
+        if is_invited:
+            from openedx.adg.lms.utils.env_utils import is_testing_environment
+            if not is_testing_environment():
+                from openedx.adg.lms.registration_extension.forms import get_registration_form_override
+                return get_registration_form_override(request, form_desc)
+
```

Comment:
To minimise core changes we usually move all the logic to our custom method and move all the checks like is_testing_env in our own method, if it is a test environment then we simply return. Lets just import it with module imports and call it here and move this logic to our own method.